src/LibraryManager/LibrariesValidator.cs

```diff
@@ -35,26 +35,34 @@ internal static class LibrariesValidator
         {
             cancellationToken.ThrowIfCancellationRequested();
 
+            // Check for valid libraries
             IEnumerable<ILibraryOperationResult> validateLibraries = await ValidatePropertiesAsync(libraries, dependencies, cancellationToken).ConfigureAwait(false);
-
             if (!validateLibraries.All(t => t.Success))
             {
                 return validateLibraries;
             }
 
-            IEnumerable<ILibraryOperationResult> expandLibraries= await ExpandLibrariesAsync(libraries, dependencies, defaultDestination, defaultProvider, cancellationToken).ConfigureAwait(false);
+            // Check for duplicate libraries 
+            if (GetDuplicateLibraries(libraries, dependencies, defaultProvider, cancellationToken) is List<string> duplicates && duplicates.Count > 0)
```

Comment:
> is List<string> duplicates && duplicates.Count > 0 [](start = 98, length = 51)

.Any()

Reply:
`duplicates` should probably be declared on the line before the if, something like:

````
IEnumerable<string> duplicates = GetDuplicateLibraries(...);
if(duplicates.Any())
{
    return new (...);
}
````

---
In reply to: [198657528](https://github.com/aspnet/LibraryManager/pull/220#discussion_r198657528) [](ancestors = 198657528)